FlightReservation/src/helpers/firebaseSignUp.js

```diff
@@ -16,3 +19,22 @@ export const signInWithNameEmailAndPassword = (name, email, password) => {
       });
   });
 };
+
+export const useGoogleConfiguration = () => {
+  useEffect(() => {
+    GoogleSignin.configure({
+      webClientId: '485144115607-mricdh0gp8vvo7911rvjc4ahb5mavbjk.apps.googleusercontent.com',
```

Comment:
this is confidential info you are compromising the project please move this to a .env file and call it to to get the value

Reply:
Supporting @DemisRincon 's comment, you can use a library that allows you to use .env files, and save your private keys.

One that I have used and it has helped me a lot is:

[react-native-dotenv](https://www.npmjs.com/package/react-native-dotenv)